Resources/public/css/theme/views/languageswitcher.css

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+
+.ez-view-languageswitcherview .ez-dropdown-list-indicator {
+    color: #333;
+}
+
+.ez-view-languageswitcherview .ez-dropdown-list-indicator:after {
+    display: inline-block;
+    content: "\E602";
+    font-family: 'ez-platformui-icomoon';
+    color: #333;
+    padding-left: 0.1em;
+    vertical-align: bottom;
+}
+
+.ez-view-languageswitcherview .ez-expandable-area {
+    border: 1px solid #ccc;
+    box-shadow: 0 1px 5px rgba(0, 0, 0, 0.3);
+    background: #fff;
+    display: block;
+    transform: scaleY(0);
+    transform-origin: 50% 0%;
+    -webkit-transform-origin: 50% 0%;
```

Comment:
after testing it, 0.3s is maybe a bit slow in this case, I think 0.2s would give a better result.